osm2gtfs/core/creator_factory.py

```diff
@@ -5,15 +5,16 @@
 from osm2gtfs.creators.feed_info_creator import FeedInfoCreator
 from osm2gtfs.creators.routes_creator import RoutesCreator
 from osm2gtfs.creators.stops_creator import StopsCreator
+from osm2gtfs.creators.schedule_creator import ScheduleCreator
 from osm2gtfs.creators.trips_creator import TripsCreator
 
 
 class CreatorFactory(object):
 
     def __init__(self, config):
         self.config = config
-        if 'selector' in config:
-            self.selector = config['selector']
+        if 'selector' in self.config.config:
+            self.selector = self.config.config['selector']
```

Comment:
config.config looks confusing. Why does this happen now?

Reply:
Because the `config` object of `Configuration` contains an attribute `config` with the processed information from the configuration file. In this PR I started giving the full `config` object to the creators, because some of them might want to use functions attached to `Configuration`, for example the newly introduced `schedule_creator` does this, to load the schedule information when needed.

Reply:
If you want to do this, you might want to rethink the naming at the same time, so people reading the code have a better change at knowing what is what.

Reply:
What about `config.data`? Or is this too generic?

Reply:
I guess `data` is fine, don't have a better idea right now. You forgot to adapt the `param` docstrings where the Configuration object is passed, btw.